corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -519,21 +519,6 @@ def hard_delete_forms(domain, form_ids, delete_attachments=True):
 
         return deleted_count
 
-    @staticmethod
-    def get_attachments_for_forms(form_ids, ordered=False):
```

Comment:
This was deleted here: https://github.com/dimagi/commcare-hq/commit/361d336e09101cf80d6ba8e0ee0d4caea1084ea9#diff-d28c016fb16e34a86ad9ce844e326789L535
It must have gotten added back when I merged with master, so I deleted it here.